README.md

```diff
@@ -16,3 +16,11 @@ This is the repository for CS3219 Group 28 PeerPrep project.
 ## Frontend
 1. Install npm packages using `npm i`.
 2. Run Frontend using `npm start`.
+
+## Question Service
+This service is used to Create, Remove, Get the interview questions in the Room page.
+1. Rename `.env.sample` file to `.env`.
+2. Create a Cloud DB URL using Mongo Atlas.
+3. Enter the DB URL created as `DB_CLOUD_URI` in `.env` file.
+4. Install npm packages using `npm i`.
+5. Run User Service using `npm start`.
```

Comment:
```suggestion
5. Run Question Service using `npm start`.
```
I think you forgot to change this haha

Reply:
Thanks for this haha, I did not save the file.